api/valor_api/backend/core/evaluation.py

```diff
@@ -489,7 +492,21 @@ def fetch_evaluations(
         dataset_names=dataset_names,
         model_names=model_names,
     )
-    return db.query(models.Evaluation).where(*expr).all()
+
+    stmt = (
+        update(models.Evaluation)
+        .returning(models.Evaluation)
+        .where(
+            and_(
+                *expr,
+                models.Evaluation.status != enums.EvaluationStatus.DELETING,
+            )
+        )
+        .values(status=enums.EvaluationStatus.DELETING)
+        .execution_options(synchronize_session="fetch")
+    )
+
+    return db.execute(stmt).scalars().all()
 
 
 def fetch_evaluation_from_id(
```

Comment:
We should add a condition to this query so that it doesn't return evaluations that are marked for deletion

Reply:
This would go for all of the get_evaluation methods too

Reply:
i'm not sure i understand-- we want this to return all the ones that were newly marked for deletion